### DISCUSSION TITLE
Skin Options

### DISCUSSION THREAD

**Discussion by L337Hun73r** [Kron4ek/minecraft-vortex-launcher#68] [Ideas]
Could it be doable to add a way of skin selection and adding custom skins to the launcher?

**Comment by Kron4ek** [comment]
So that custom skins would only be visible for you or for other players too? The former is easily doable, the latter is much harder to implement.

**Reply by L337Hun73r** [reply]
I, for one, just want it for myself. I would appreciate being the only confirmed male AND fair skinned square faced guy once more :P 

To be honest, Mojang is getting more woke every single day. They darkened Steve's skin color, added 7 extra ungendered persons of color character skins and such. So yeah...

Other that that, does V Launcher have multiplayer options for skins to be visible to other people? If not, when you do implement the multiplayer, maybe you could make the skins visible to others too. And even if there is a way to play multiplayer, there's no need to rush, skins are not a game changer anyway.

**Reply by cgarz** [reply]
Yea Mojang is a joke these days. I feel ashamed having even just once considered buying the game. Even though that was before M$ bought them.

Anything that helps undo their virtue signalling would be most welcome.

**Comment by jedai86** [comment]
You can use one of the mods that adds custom skins, саpes and other:
https://www.curseforge.com/minecraft/mc-mods/capes  
https://www.curseforge.com/minecraft/mc-mods/customskinloader
https://www.curseforge.com/minecraft/mc-mods/cosmetics

**Comment by Dargus777** [comment]
I have a question, how to install mods?
and is it possible to make my skin visible to other players in my world?
I will be grateful if you answer, download the launcher, but there are no instructions on how to install it, thank you in advance.

**Reply by Kron4ek** [reply]
> I have a question, how to install mods?

You need to install either [Forge](https://files.minecraftforge.net/net/minecraftforge/forge/index_1.19.4.html) or [Fabric](https://fabricmc.net/use/installer/).

To install Forge:
1. Download its installer and run it
2. Specify the game directory, press "OK"

To install Fabric:
1. Download its installer and run it
2. Select the game version, specify the game directory, press "Install"
3. Download [Fabric API](https://beta.curseforge.com/minecraft/mc-mods/fabric-api/files) and put it to the directory named "mods", which is inside the game directory (if it does not exist, create it)

After the installation, in the launcher you should see Fabric or Forge in the version list and will be able to select them.

Now to install mods simply download them and put them to the mods directory.

**Reply by L337Hun73r** [reply]
My original query was for having the V-Launcher as independent as can be from Forge and Fabric, but this might work too as a temporary solution.
I want my V-Launcher to be the best one around as it's loyal "customer" since early days :P

**Reply by Dargus777** [reply]
thanks for the answer. :)
by the way, I figured out the mods. As for the skins, you can simply install the "more player models" mod and then you can change the skins to any one you want, and it will be visible to all other players who also have this mod installed.
